src/app/components/ScrollToHash.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
```

Comment:
I suggested putting each and every component to a folder. The one, which needs to be reused in the app will be exposed through index.ts file. Inside the folder you can put styles, tests, etc. Consider this folder as a black box for the rest of the app, it exposes only what is allowed to app to use.
You can think of a such structure:
```
component:
- component.tsx
- component.css
- component.test.tsx
- lib.ts

```

Reply:
You need to export all components from the components folder. Same applied to features.

Reply:
I asked this question in two places, and got answers in both. Thank You)